src/components/CookieBanner/CookieBanner.test.js

```diff
@@ -0,0 +1,29 @@
+import Enzyme, { mount } from 'enzyme';
+import CookieBanner from './CookieBanner';
+import Adapter17 from '@wojtekmaj/enzyme-adapter-react-17';
```

Comment:
i think are we using react v17. but why adapter 16 is also installed? hmmm

Reply:
I have removed the reference to adapter 16